apps/files_sharing/src/views/SharingDetailsTab.vue

```diff
@@ -727,15 +734,24 @@ export default {
 	beforeMount() {
 		this.initializePermissions()
 		this.initializeAttributes()
-		console.debug('shareSentIn', this.share)
-		console.debug('config', this.config)
+		logger.debug('Share object received', { share: this.share })
+		logger.debug('Configuration object received', { config: this.config })
 	},
 
 	mounted() {
 		this.$refs.quickPermissions?.querySelector('input:checked')?.focus()
 	},
 
 	methods: {
+		async getNode() {
+			const node = { path: this.path }
+			try {
+				this.node = await fetchNode(node)
+				logger.info('Fetched Node:', this.node)
```

Comment:
nitpick: Prevent the context from being merged with the node:

```suggestion
				logger.debug('Fetched Node:', { node: this.node })
```